lib/tasks/demo_data_for_development.rake

```diff
@@ -37,27 +37,39 @@ namespace :data do
         create(:registration, user: user2, conference: conference)
     end
 
-    conference = create(:full_conference, title: 'Jangouts Demo Conference', short_title: 'jangouts', start_hour: 15, start_date: 7.days.ago, description: "This is a full conference demo instance happened in the past. It contains open cfp, venue/rooms, submitted talks by multiple speakers, partly confirmed talks and multiple schedules.")
+    # This is a full conference demo instance that has already happend.
+    # By full conference it means all basic information about conference is already set.
+    conference = create(:full_conference, title: 'Jangouts Demo Conference', short_title: 'jangouts', start_date: 7.days.ago, start_hour: 15, end_hour: 20, description: 'This is a full conference demo instance happened in the past. It contains open cfp, venue/rooms, submitted talks by multiple speakers, partly confirmed talks and multiple schedules.')
     generate_program conference
-    conference.program.cfp.update_attributes(start_date: 4.days.ago, end_date: 2.days.ago)
-    conference.update_attributes(end_date: 1.day.ago)
-    conference.registration_period.update_attributes(start_date: 9.days.ago, end_date: 8.days.ago)
+    conference.program.cfp.update_attributes!(start_date: 4.days.ago, end_date: 2.days.ago)
+    conference.update_attributes!(end_date: 1.day.ago)
+    conference.registration_period.update_attributes!(start_date: 9.days.ago, end_date: 8.days.ago)
 
-    conference = create(:full_conference, title: 'Open Source Event Manager Demo Conference', short_title: 'osemdemo', start_hour: 8, description: "This is a full conference demo instance happening in the future. It contains open cfp, venue/rooms, submitted talks by multiple speakers, partly confirmed talks and multiple schedules.")
+    # This is a full conference demo instance that will happen in the future.
+    # By full conference it means all basic information about conference is already set.
+    conference = create(:full_conference, title: 'Open Source Event Manager Demo Conference', short_title: 'osemdemo', start_date: 2.days.from_now, end_date: 6.days.from_now, start_hour: 8, end_hour: 20, description: 'This is a full conference demo instance happening in the future. It contains open cfp, venue/rooms, submitted talks by multiple speakers, partly confirmed talks and multiple schedules.')
     generate_program conference
 
+    # This is a conference demo instance that will happen in the future.
+    # No information about this conference is set by default.
```

Comment:
I am confused about what that means. It seems we are setting a registration period for our conference, as well as some events.
Perhaps the conference has a registration period and unscheduled events?